jobs/cronjob-truncate-cassandra-tag-metrics.yml

```diff
@@ -0,0 +1,198 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: cronjob-truncate-cassandra-tag-metrics
+  annotations:
+    description: Scheduled Task to Truncate Cassandra Tags Index
+    iconClass: icon-shadowman
+    tags: management,cronjob,truncate,cassandra
+objects:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: ${JOB_SERVICE_ACCOUNT}
+    labels:
+      template: "cronjob-truncate-cassandra-tag-metrics"
+- kind: ClusterRole
+  apiVersion: v1
+  metadata:
+    name: cassandra-cleaner
+    labels:
+      template: "cronjob-truncate-cassandra-tag-metrics"
+  rules:
+  - apiGroups:
+    - ""
+    attributeRestrictions: null
+    resources:  
+    - pods
+    - pods/exec
+    - replicationcontrollers
+    - replicationcontrollers/scale
+    verbs:
+    - get
+    - list
+    - create
+    - update
+- apiVersion: authorization.openshift.io/v1
+  groupNames: null
+  kind: ClusterRoleBinding
+  metadata:
+    name: cassandra-cleaners
+    labels:
+      template: "cronjob-truncate-cassandra-tag-metrics"
+  roleRef:
+    name: cassandra-cleaner
+    namespace: ${NAMESPACE}
```

Comment:
References to `ClusterRole`'s cannot be namespaced

Reply:
Sorry :(